src/invidious/videos/parser.cr

```diff
@@ -107,7 +107,7 @@ def extract_video_info(video_id : String, proxy_region : String? = nil)
     # decrypted URLs and maybe fix throttling issues (#2194). See the
     # following issue for an explanation about decrypted URLs:
     # https://github.com/TeamNewPipe/NewPipeExtractor/issues/562
-    client_config.client_type = YoutubeAPI::ClientType::Android
+    client_config.client_type = YoutubeAPI::ClientType::AndroidTestSuite
     new_player_response = try_fetch_streaming_data(video_id, client_config)
   elsif !reason.includes?("your country") # Handled separately
     # The Android embedded client could help here
```

Comment:
We do not need to keep AndroidScreenEmbed, it does not work anymore.

Reply:
Are you sure? I haven't tried that, so I can't confirm nor deny.

Reply:
I have tried it, it results to the same exact issue, you get "please update your app" video error.